pdf-view-restore.el

```diff
@@ -65,8 +65,15 @@
   "Restore page."
   (when (derived-mode-p 'pdf-view-mode)
     ;; This buffer is in pdf-view-mode
-    (let ((page (pdf-view-restore-get-page)))
-      (when page (pdf-view-goto-page page)))))
+    (let ((page (pdf-view-restore-get-page))
+          (pdf-pages (pdf-info-number-of-pages)))
+      (when page
+        (if (<= page pdf-pages)
```

Comment:
If the page value comes from the pdf, when would it exceed the total number of pages?

Reply:
I simply created this patch on directly on github (that's also why I did not squash/extend the commits), I don't remember, but probably I expected that the commit message would also get used as the PR message. Anyway, in the first commit message I wrote that it happened somehow, possibly because I have multiple PSFs with the same name (i.e. main.pdf, form different latex projects).